docs/adapting-new-services.md

```diff
@@ -159,6 +157,27 @@ Services that provide aggregation, e.g., concatenation for CONCISE, require that
 available when they are run. Harmony infers this from the `operations` field in the associated step.
 Currently the only supported aggregation operation is `concatenate`.
 
+There are limits to the number of files an aggregating service can process as well as the total number
+of bytes of all combined input files. To support larger aggregations Harmony can partition the output
+files from one service into batches to be passed to multiple invocations of the next (aggregating)
+step. Whether or not a service should have its input batched, the maximum number of input files to include
+in each batch, and total combined file sizes to allow can be set in the aggregating service's step
+definition using the `is_batched`, `max_batch_inputs`, and `max_batch_size_in_bytes` flags.
+
+Note that there is a system imposed upper limit on `max_batch_size_in_bytes`. 
+
+The following `steps` entry is an example one might use for an aggregating service:
+
+```yaml
+steps:
+      - image: !Env ${QUERY_CMR_IMAGE}
+      - image: !Env ${EXAMPLE_AGGREGATING_SERVICE_IMAGE}
+        is_batched: true
+        max_batch_inputs: 100
+        max_batch_size_in_bytes: 2_000_000_000
```

Comment:
I know this was an issue with our environment variable configuration to use `_` in a number. Does this work in services.yml? Probably just change it to 2000000000 to not have to worry about it.

Reply:
Done.